zip-draft-XXXX.rst

```diff
@@ -0,0 +1,269 @@
+
+.. The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+.. defined in the Zcash protocol specification [#protocol-networks]_.
+
+.. The term "full validator" in this document is to be interpreted as defined in
+.. the Zcash protocol specification [#protocol-blockchain]_.
+
+.. The terms below are to be interpreted as follows:
+
+.. {Term to be defined}
+..   {Definition.}
+.. {Another term}
+..   {Definition.}
+
+
+.. |percentage| replace:: 50%
+
+
+
+Abstract
+========
+
+This ZIP proposes the allocation of |percentage| of the Zcash block reward,
+post-November 2024 halving, to a decentralized "lockbox." Currently, 80% of the
+block reward goes to miners, while 20% is distributed among the Major Grants
+Fund, Electric Coin Company (ECC), and the Zcash Foundation (ZF). If no changes
+are made, this 20% dev fund will expire, resulting in the entire block reward
+going to miners, leaving no block-reward funds for essential protocol
+development, security, marketing, or legal expenses.
+
+The proposed lockbox addresses significant issues observed with [#zip-1014]_,
+such as regulatory risks, inefficiencies in funding organizations instead of
+projects, tax compliance inefficiencies, and centralization. By disbursing
+funds through a decentralized mechanism, resembling a DAO, this ZIP aims to
+ensure community involvement and efficient, project-specific funding. This
+approach is intended to improve regulatory compliance, reduce inefficiencies,
+simplify tax compliance, and enhance the decentralization of Zcash's funding
+structure.
+
+
+Motivation
+==========
+
+As of now, the Zcash block reward is allocated with 80% going to miners, and
+the remaining 20% distributed among the Major Grants Fund (8%), Electric Coin
+Company (ECC) (7%), and the Zcash Foundation (ZF) (5%). This funding structure
+supports various essential activities such as protocol development, security,
+marketing, and legal expenses. However, this model will expire in November
+2024, leading to the entire block reward being allocated to miners if no
+changes are made. This expiration poses several significant issues:
+
+1. **Regulatory Risks**: The current model, as observed with ZIP-1014, involves
+   direct funding of US-based organizations. This can potentially attract
+   regulatory scrutiny from entities such as the SEC, posing legal risks to the
+   Zcash ecosystem.
+
+2. **Funding Inefficiencies**: Directly funding organizations rather than
+   specific projects leads to inefficiencies. Organizations may accumulate
+   excess funds while specific projects may not receive adequate resources. A
+   more granular approach, funding projects directly, would be more efficient.
+
+3. **Tax Compliance Complications**: The current model introduces tax
+   inefficiencies, as 501(c)(3) organizations must treat received block rewards
+   as income, leading to significant tax liabilities. This requires selling
```

Comment:
This is likely to be the case for any entity that ultimately receives the block rewards in the future.

Reply:
entity or individual yes; but perhaps not if it's one multisig to another? Eventually an entity or individual will have some tax liability but this proposal could put it farther away from the protocol layer I think.